calviacat/catalog.py

```diff
@@ -417,7 +418,11 @@ def cal_color(self, matched, m_inst, filt, color, C=None,
             fitting.LinearLSQFitter(), sigma_clip)
 
         i = np.isfinite(dm) * ~dm.mask
-        line, fit = fitter(model, cindex[i], dm[i])
+        if astropy_version[0] > 3 or (astropy_version[0] == 3 and astropy_version[1] >= 1):
+            # Return order changed in astropy 3.1 (http://docs.astropy.org/en/stable/changelog.html#id10)
+            fit, line = fitter(model, cindex[i], dm[i])
+        else:
+            line, fit = fitter(model, cindex[i], dm[i])
```

Comment:
The docs say it returns a mask, so the new code should be `fit, mask = fitter(...`.  We don't use it either way, but I'd rather have the right variable name here.

Reply:
Updated code and comment